### PR TITLE
Update README.md with V5 release notes

### PR DESCRIPTION
**Pull request by ab**
**Description:**
Add info about the v5 release to the README.md

Previously, users would think that v4 was the latest version if the only skimmed the bold headlines in the README.

### CODE REVIEW

**Review by RichTwicz — APPROVED** [review]
Ii

**Review by viniciusgabrielfo — APPROVED** [review]

**Comment by ab** [comment]
I think this needs review by @actions/setup-actions-team listed in CODEOWNERS, but that's a private team. Maybe @chenrui333 could point to who can review + merge this?

**Review by aparnajyothi-y — APPROVED** [review]

**Review by priyagupta108 — APPROVED** [review]

**Review by HarithaVattikuti — APPROVED** [review]

**Review by Natthachai0434 — COMMENTED** [review]
git add .
git commit -m

**Pull request merged**
